src/main/resources/config/application-dev.yml

```diff
@@ -163,9 +163,16 @@ application:
   frontend:
     sentry-project-id:
     google-analytics-project-id:
+    recaptcha-project-id:
+    recaptcha-site-key:
+    recaptcha-threshold:
   aws:
     s3-access-key:
     s3-secret-key:
     s3-region:
   github-token:
   db-read-only: false # Certain endpoints will be disabled when readonly is set to true
+  recaptcha:
+    project-id:
+    site-key:
+    threshold:
```

Comment:
Are these required? Would application run without these defined here?

Reply:
Yes these are required because the `RecaptchaProperties` are used in `CreateAssessment` to load the site key and project ID